src/assignment/components/AssignmentMetaDataFormEditable.tsx

```diff
@@ -56,6 +56,19 @@ const AssignmentMetaDataFormEditable: FC<AssignmentMetaDataFormEditableProps> =
 		}));
 	};
 
+	const filterSubjects = (subject: Avo.Lom.LomField & { related?: string[] }) => {
+		const selectedEducationLevels = (assignment.loms || []).filter(({ lom }) => {
+			return lom?.scheme === 'https://w3id.org/onderwijs-vlaanderen/id/structuur';
```

Comment:
use an enum value for this since it is used multiple times in the codebase already